app/controllers/reviews_controller.rb

```diff
@@ -43,4 +48,14 @@ def destroy
   def review_params
     params.require(:review).permit(:body, :rating, :art_id)
   end
+
+  def authenticate_user
+    if !user_signed_in?
+      raise_error
+    end
+  end
+
+  def raise_error
+    raise ActionController::RoutingError.new("Not Found")
+  end
```

Comment:
These could then be removed if these are only on the user model. Also don't forget unit tests for these as well.